src/quo2/components/avatars/group_avatar/view.cljs

```diff
@@ -7,24 +7,27 @@
 
 (def sizes
   {:icon      {:small  12
+               28      16
```

Comment:
how possible would it be to update the other uses here and fix across the app so we're not mixing semantic names with integers? 🤔

Reply:
As I wrote in this PR description:

> So I implemented them in order to have this PR working and pass design review, but these components should be checked in the same way I did it with this one.

I meant, we need to refactor, add the missing sizes and fix the API of these components.

Reply:
Thanks, my bad for missing that!